### PR TITLE
Adding challenge questions to the database

### PR DESCRIPTION
**Pull request by sahmedkh**
This PR contains the challenges and the plan template

### CODE REVIEW

**Comment by sahmedkh** [comment]
@mzhubail and @Ma-hawaj if y'all could look into the entry I made and confirm if I am following the expected format, I would appreciate it.

**Comment by Ma-hawaj** [comment]
Seems Good.

File: QuestionScript/questionsNew.json [hunk 1]
```diff
@@ -3963,5 +3963,4899 @@
...
+      "context": {
+        "S": "Listen to the business interview to practise and improve your listening skills."
+      },
+      "contextAudio": {
+        "S": "listeningChallengeB2-1.mp3"
+      },
+      "tasks": {
+        "L": [
+          {
+            "M": {
```
Comment (Ma-hawaj) [inline QuestionScript/questionsNew.json:3979]: Questions array need to be removed

File: QuestionScript/questionsNew.json [hunk 2]
```diff
@@ -3963,5 +3963,4899 @@
         }
       }
     }
+  ],
+
+  "challenge-B2-listening": [
+    {
```
Comment (Ma-hawaj) [inline QuestionScript/questionsNew.json:3969]: Is there a title for every  challenge or should we make it optional

**Comment by Ma-hawaj** [comment]
You can see the problem of Questions array in the last challenge in sample challenge file

**Comment by sahmedkh** [comment]
@Ma-hawaj, I fixed the questions array problem, but why did the `Challenge` type change? It is different from what we agreed upon initially.

**Comment by Ma-hawaj** [comment]
@sahmedkh I already made the additions optional

The level and type can be removed since they will be in the PK and the title would be nice addition sara added it for the frontend.

**Comment by Ma-hawaj** [comment]
I think the following challenge type is what we want:

```
export type Challenge = {
  contextTitle?: string;
  context: string;
  contextAudio?: string;
  tasks: LRQuestion[];
};
```

**Review by xvbilalvx — APPROVED** [review]

**Pull request merged**